### DISCUSSION TITLE
[Feature] Slideshow for assets in an album

### DISCUSSION THREAD

**Discussion by radh21301** [immich-app/immich#2767] [Feature Request]
### The feature

Maybe I am the only one, but I use the slideshow feature in google photos quite a lot. I add photos to a certain album and start the slideshow -> I do this on an old tablet which essentially transforms into a digital photo frame.

_Nice to have:_

- Some control over how frequently the images change (GPhotos has 5 seconds or so which is too fast for my use case. But technically the feature was not intended for that, so that is okay)
- Maybe randomizing the order of the images? This is not so important I think.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by jrasm91** [comment]
This is implemented on the web a few releases back
